.github/workflows/ci-cd-pull-request.yml

```diff
@@ -21,13 +21,13 @@ jobs:
       - name: Set up pnpm
         uses: pnpm/action-setup@v2
         with:
-          version: 7.28.0
+          version: 8.2.0
```

Comment:
If we have this set, maybe we should add `packageManager` to root `package.json`?  just to make sure we are not conflicting, because some are still using pnpm7 locally

Reply:
[It's already there](https://github.com/graphprotocol/docs/pull/345/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R4), or do you mean something else?

Reply:
oh oops, i missed that 🤦